gslabModel/R/loadFromDisk.R

```diff
@@ -0,0 +1,23 @@
+#' Load a \code{ModelData} object from files saved using \code{saveToDisk} 
+#' @description Data are read from a \code{.csv} file and other fields are read from a \code{.rds}
+#' file. The group variable with name \code{group_export} is deleted.
+#' @param obj A ModelData object, typically an empty one.
+#' @param directory location of files to be loaded.
+#' @param name name of files to be loaded.
+#' @param collapseArrayVars whether array variables are collapsed. Default is collapse.
+#' @export
+#' 
+loadFromDisk = function(obj, directory, name, collapseArrayVars = 1) {
+    tempObj <- readRDS(paste(directory, "/", name, ".rds", sep = ""))
```

Comment:
Good idea!

A small issue: I implement this and in the test,
```
...
saveToDisk(a, ".", "myObj", 12)
b <- loadFromDisk(".", "myObj")
```
 However, `expect_equal(a, b, tolerance = 1e-12)` fails. It says 
> Class definitions are not identical

But when I check the class name, class definition (`getClassDef(class(a))` and `getClassDef(class(b))`), and the value of every filed, they all match. So I'm confused why the test fails. Currently I switch to check each field of the two objects. It would be great if you can take a try to see where they differ.